pkg/registry/populator.go

```diff
@@ -255,7 +255,7 @@ func (i *DirectoryPopulator) loadManifestsSemver(bundle *Bundle, skippatch bool)
 	}
 
 	if err := i.loader.AddBundleSemver(updatedGraph, bundle); err != nil {
-		return fmt.Errorf("error loading bundle into db: %s", err)
+		return fmt.Errorf("error loading bundle %s into db: %s", bundle.Name, err)
```

Comment:
It seems your intention here to print the error message then it should be `err.Error()`?
Same applied for all below ones as well.

Reply:
The fmt will call the string interface for the errors, see: https://pkg.go.dev/fmt#hdr-Printing:

> If an operand implements the error interface, the Error method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any).

Because of this, before this PR we get the errors that came from the SQL. The problem is that without context, we have no additional info to try troubleshooting.